pages/05.android/06.solving-problems/06.low-level-settings/docs.en.md

```diff
@@ -31,4 +31,16 @@ We united several settings under this category: _pref.https.apps.exclusions_, _p
 
 **pref.vpn.ipv6.bypass**
 
-Nowadays, more and more often SMS and MMS are sent via network, and when they are, non-standard IPv6 network is often used. When we try to filter this traffic, it is not extremely unusual that we interfere with sending and even receiving MMS. If you suspect this is your case, we recommend checking the **pref.vpn.ipv6.bypass** checkbox or adding the app responsible for MMS managing to **pref.net.exclusions**.
\ No newline at end of file
+Nowadays, more and more often SMS and MMS are sent via network, and when they are, non-standard IPv6 network is often used. When we try to filter this traffic, it is not extremely unusual that we interfere with sending and even receiving MMS. If you suspect this is your case, we recommend checking the **pref.vpn.ipv6.bypass** checkbox or adding the app responsible for MMS managing to **pref.net.exclusions**.
+
+**pref.samsungpay.autopause.enable**
```

Comment:
We need a Korean translation here as well.

Also, it'd be better to have a GIF file demonstrating how this feature works.